core/src/main/java/greencity/dto/user/UserManagementDto.java

```diff
@@ -0,0 +1,36 @@
+package greencity.dto.user;
+
+import greencity.constant.ValidationConstants;
+import greencity.entity.enums.UserStatus;
+import lombok.*;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+public class UserManagementDto {
+    @NotNull
+    private Long id;
+
+    @NotBlank(message = ValidationConstants.EMPTY_USERNAME)
+    @Size(
+        min = ValidationConstants.USERNAME_MIN_LENGTH,
+        max = ValidationConstants.USERNAME_MAX_LENGTH,
+        message = ValidationConstants.INVALID_USERNAME_LENGTH)
+    private String name;
+
+    @Email(message = ValidationConstants.INVALID_EMAIL)
```

Comment:
Those messages are not i18n, but the application is intended to support at least 3 languages. Was the problem with error messages translation solved?

Reply:
I did not see any problems with this, and also as far as I remember we decided to do our admin panel only in English. 
![image](https://user-images.githubusercontent.com/45639641/90753627-824fac80-e2e1-11ea-9171-e20ff6b217b2.png)